sdks/python/build.gradle

```diff
@@ -250,6 +250,18 @@ task hdfsIntegrationTest(dependsOn: 'installGcpTest') {
   }
 }
 
+task flinkCompatibilityMatrix() {
+  dependsOn 'setupVirtualenv'
+  dependsOn ':beam-sdks-python-container:docker'
+  dependsOn ':beam-runners-flink_2.11-job-server:jar'
+  doLast {
+    exec {
+      executable 'sh'
+      args '-c', ". ${envdir}/bin/activate && pip install -e . && python -m apache_beam.runners.portability.flink_runner_test " + project(":beam-runners-flink_2.11-job-server:").shadowJar.archivePath
```

Comment:
FYI, you can just use string interpolation here instead of concatenating. e.g. `....flink_runner test ${project("...").shadowJar.archivePath}". IIRC, Groovy even handles nested quotes correctly when used this way.

Reply:
Oh, nice. Done.